cfme/fixtures/service_fixtures.py

```diff
@@ -28,13 +28,13 @@ def catalog_item(provider, provisioning, vm_name, dialog, catalog):
     return catalog_item
 
 
-def create_catalog_item(provider, provisioning, vm_name, dialog, catalog, console_template=None):
+def create_catalog_item(provider, provisioning, vm_name, dialog, catalog, console_test=False):
```

Comment:
**OPTIONAL:** As long as you are here would you mind creating a doc string for this function.

Reply:
**QUESTION:** I'm just ignorant here, so educate me please.   How does changing this to console_test fix the problem?   Also, where is console_test coming from?    I don't see it being imported.

Reply:
Yes I'll add the doc string. So the console_test is just a rename of variable to remove ambiguity. 
- console_test : Variable to tell if test is console test or not. True or False. By default False.
- console_template: it is a fixture in fixtures/provider.py which will try to fetch console template information from provider data from cfme_data.yaml.

Changing that fixes the problem because, previously for providers such as SCVMM, GCE, Azure, etc. Test will try to execute console_template fixture function and that will fail silently if the console_template field is not found in the provider data. That will cause skipping of the test. Which now won't happen as console_template fixture will not be executed for all providers but only during test with provider which really have console template available on it (and in cfme_data.yaml).

Does that make sense?

Reply:
Mostly.   Sounds like some of this information could be in the docstring, essentially mentioning why you use console_test.    I still don't see where it is imported though.

Reply:
Sure. So I will add the doc string once @sshveta confirms this solution works for her. And again, the *console_test* is just a parameter to the function and not anything external. Hence nothing imported anywhere.

Reply:
Brain is slow, will seek more caffeine.   Got it.

Reply:
I got confused by all the stuff that auto-vivifies in this pytest environment.   It's not all magic but it feels that way sometimes.

Reply:
Yes a lot of time I have trouble figuring out what is direct parameter and what is fixture or woodoo😜 won't blame u

Reply:
Works fine @kedark3 :+1: